src/ImageGallery.react.jsx

```diff
@@ -272,6 +272,7 @@ export default class ImageGallery extends React.Component {
               className={this.props.server ? 'loaded' : null}
               src={item.original}
               alt={item.originalAlt}
+              name={item.originalName}
```

Comment:
The name attribute has been deprecated since HTML 4.01 (1999) I don't think any spiders should index the name attribute.